### PR TITLE
[ADVAPP-918]: Cleanup FeatureFlag: AddUrlToThemeSettingsFeature

### PR DESCRIPTION
**Pull request by kandarp-canyon**
### Ticket(s) or GitHub Issue

- https://canyongbs.atlassian.net/browse/ADVAPP-918

### Technical Description

> removed all feature flag related files and cleanup the code.

### Any deployment steps required?

> No.

### Are any Feature Flags and/or Data Migrations that can eventually be removed Added?

> No

_______________________________________________

### CODE REVIEW

**Review by Orrison — APPROVED** [review]

**Review by Orrison — CHANGES_REQUESTED** [review]
Looks like there is a small merge conflict that just needs to be resolved.

File: app-modules/theme/src/Filament/Pages/ManageBrandConfigurationSettings.php [hunk 1]
```diff
@@ -178,8 +176,7 @@ public function form(Form $form): Form
                                     ->visible(fn (Get $get) => $get('is_custom_link_url_enabled'))
                                     ->columnSpanFull(),
                             ]),
-                    ])
-                    ->visible(fn () => AddBrandedWebsitesToThemeSettingsFeature::active()),
+                    ]),
```
Comment (Orrison) [inline app-modules/theme/src/Filament/Pages/ManageBrandConfigurationSettings.php:179]: This doesn't seem to be the Feature Flag mentioned in the title of this ticket? Should this have been removed?

**Review by Orrison — CHANGES_REQUESTED** [review]

File: app-modules/theme/src/Filament/Pages/ManageBrandConfigurationSettings.php (hunk 1 above)
Comment (kandarp-canyon) [reply app-modules/theme/src/Filament/Pages/ManageBrandConfigurationSettings.php]: No, Added the feature flag again.

File: app-modules/theme/src/Filament/Pages/ManageBrandConfigurationSettings.php [hunk 2]
```diff
@@ -177,9 +176,9 @@ public function form(Form $form): Form
                                     ->url()
                                     ->visible(fn (Get $get) => $get('is_custom_link_url_enabled'))
                                     ->columnSpanFull(),
-                            ]),
-                    ])
-                    ->visible(fn () => AddBrandedWebsitesToThemeSettingsFeature::active()),
+                            ])
+                            ->visible(fn () => AddBrandedWebsitesToThemeSettingsFeature::active()),
+                    ]),
```
Comment (Orrison) [inline app-modules/theme/src/Filament/Pages/ManageBrandConfigurationSettings.php:181]: This doesn't look to have been placed back into the right spot.

**Review by Orrison — CHANGES_REQUESTED** [review]

**Review by Orrison — APPROVED** [review]

**Pull request merged**